csuntechlab/checkpoint
# Check 137 - Create Home page with authenticated Admins

# Description
- Created admin page
- Added route /admin
- Added jest test

## Type of change

Please delete options that are not relevant.



# How Has This Been Tested?
- Start up your environment `docker-compose up -d`
- Navigate to [http://localhost:8080/#/admin](http://localhost:8080/#/admin)
- Verify page looks like the mock-ups

# Checklist:
